.travis.yml

```diff
@@ -5,41 +5,61 @@ notifications:
-    - ASTYLE_ONLY="OFF" COMPILER="clang-5.0" MOAB_VERSION="5.1.0"  BUILD_GEANT4="ON"
-    # vary moab version
-    - ASTYLE_ONLY="OFF" COMPILER="gcc-7"     MOAB_VERSION="master" BUILD_GEANT4="ON"
-    - ASTYLE_ONLY="OFF" COMPILER="clang-5.0" MOAB_VERSION="master" BUILD_GEANT4="ON"
-    # vary compiler
-    - ASTYLE_ONLY="OFF" COMPILER="gcc-4.8"   MOAB_VERSION="5.1.0"  BUILD_GEANT4="OFF"
-    - ASTYLE_ONLY="OFF" COMPILER="gcc-5"     MOAB_VERSION="5.1.0"  BUILD_GEANT4="OFF"
-    - ASTYLE_ONLY="OFF" COMPILER="gcc-6"     MOAB_VERSION="5.1.0"  BUILD_GEANT4="OFF"
-    - ASTYLE_ONLY="OFF" COMPILER="clang-4.0" MOAB_VERSION="5.1.0"  BUILD_GEANT4="OFF"
 services:
   - docker
+env:
+  - UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=true  COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=5.1.0
+  - UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=5.1.0
+  - UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=5.1.0
+  - UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=5.1.0
+  - UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=5.1.0
+  - UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=master
+  - UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=master
+  - UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=master
+  - UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=master
+matrix:
+  fast_finish: true
+  allow_failures:
+    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=master
+    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=master
+    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=master
+    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=master
 before_install:
-  - sudo docker pull cnerg/dagmc-ci-ubuntu-16.04
-  # get a container id
-  - sudo docker run cnerg/dagmc-ci-ubuntu-16.04 /bin/bash -c "cd /root"
-  - commit_id=$(sudo docker ps -l | tail -n1 | awk '{print $1}')
-  # make the build directory
-  - sudo docker run cnerg/dagmc-ci-ubuntu-16.04 /bin/bash -c "mkdir -p /root/build/$COMPILER/DAGMC-moab-$MOAB_VERSION"
-  - commit_id=$(sudo docker ps -l | tail -n1 | awk '{print $1}')
-  - sudo docker commit $commit_id cnerg/dagmc-ci-ubuntu-16.04
-  # copy the repo in the docker instance
-  - sudo docker cp ../DAGMC $commit_id:/root/build/$COMPILER/DAGMC-moab-$MOAB_VERSION
-  - commit_id=$(sudo docker ps -l | tail -n1 | awk '{print $1}')
-  - sudo docker commit $commit_id cnerg/dagmc-ci-ubuntu-16.04
+  # Set some local variables
+  - docker_image=ljacobson64/dagmc-ci-ubuntu-${UBUNTU_VERSION}
```

Comment:
this needs to change to `cnerg/damci-ubuntu-${UBUNTU_VERSION}`

(I know you meant to, this is just to avoid forgetting it:))